DESCRIPTION

```diff
@@ -30,7 +30,8 @@ Imports:
     htmlwidgets,
     jsonlite,
     withr,
-    httpuv
+    httpuv,
+    rstudioapi (>= 0.8.0.9002)
```

Comment:
I think the DESCRIPTION file also needs this:

```
Remotes:
    rstudio/rstudioapi
```

At least until a new version of rstudioapi is on CRAN. I'll just make the change.